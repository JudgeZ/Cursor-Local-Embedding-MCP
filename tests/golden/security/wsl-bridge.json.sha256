e9f6996ee8934ed7fa1ca70ee8d088d9e9af52fd22744195c3f5d3219dc2562c  wsl-bridge.json
