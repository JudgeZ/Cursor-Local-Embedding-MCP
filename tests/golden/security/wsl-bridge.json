{
  "capture_id": "wsl-bridge-zaevrynth-ci",
  "generated_at": "2024-03-18T20:20:12Z",
  "proxy": {
    "host": "127.0.0.1",
    "port": 5173
  },
  "windows_client": {
    "computer": "WIN-CI",
    "transport_user": "ZA-LAB\\dpapi.test"
  },
  "wsl_peer": {
    "distro": "Ubuntu-22.04",
    "kernel": "5.15.133.1-microsoft-standard-WSL2"
  },
  "tls": {
    "negotiated_version": "TLS1.3",
    "cipher": "TLS_AES_128_GCM_SHA256",
    "dpapi_session_key": "7c9e6679-7425-40de-944b-e07fc1f90ae7"
  }
}
